demo/lightning/checkpoint/multinode/README.md

```diff
@@ -0,0 +1,40 @@
+# Checkpoint Demo for PyTorch Lightning
+
+The code in this folder provides a training demo for multi node checkpointing with PyTorch Lightning. This demo is under development.
+
+## Limitations
+* Custom Strategy needs to be implemented and passed to the trainer.
```

Comment:
We should probably hyperlink to pytorch strategy docs here (i think this is it but please double check https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.strategies.Strategy.html)

Reply:
Good idea. Done